docs/adr/adr-007-minimal-changes-to-tendermint.md

```diff
@@ -0,0 +1,187 @@
+ - Introduction (https://github.com/celestiaorg/celestia-core/pull/144)
+ - Initial integration (https://github.com/celestiaorg/celestia-core/pull/152)
+ - Custom Multihash (https://github.com/celestiaorg/celestia-core/pull/155)
+ - Puting data during proposal (https://github.com/celestiaorg/celestia-core/pull/178)
+ - Module name (https://github.com/celestiaorg/celestia-core/pull/151)
+ - Update rsmt2d (https://github.com/celestiaorg/celestia-core/pull/290)
+ - Make plugin a package (https://github.com/celestiaorg/celestia-core/pull/294)
+
+#### Adding DAH to Stuff
+ - Adding DAH to Proposal (https://github.com/celestiaorg/celestia-core/pull/248/files)
+ - Blockmeta (https://github.com/celestiaorg/celestia-core/pull/372)
+
+#### Embedding DAS
+ - GetLeafData (https://github.com/celestiaorg/celestia-core/pull/212)
+ - RetrieveBlockData (https://github.com/celestiaorg/celestia-core/pull/232)
+ - ValidateAvailability (https://github.com/celestiaorg/celestia-core/pull/270)
+ - Prevent double writes to IPFS (https://github.com/celestiaorg/celestia-core/pull/271)
+ - Stop Pinning (https://github.com/celestiaorg/celestia-core/pull/276)
+ - Rework IPFS Node (https://github.com/celestiaorg/celestia-core/pull/334)
+ - Refactor for putting the block (https://github.com/celestiaorg/celestia-core/pull/338)
+ - Config for IPFS node (https://github.com/celestiaorg/celestia-core/pull/340)
+ - IPLD Dag instead of CoreAPI (https://github.com/celestiaorg/celestia-core/pull/352)
+ - Adding the DAG to the blockstore (https://github.com/celestiaorg/celestia-core/pull/356)
+ - Saving and Loading using IPFS (https://github.com/celestiaorg/celestia-core/pull/374)
+ - Manual Providing (https://github.com/celestiaorg/celestia-core/pull/375)
+ - Refactor node provider (https://github.com/celestiaorg/celestia-core/pull/400)
+ - DAS in light client workaround (https://github.com/celestiaorg/celestia-core/pull/413)
+
+
+
+#### BlockID and PartSetHeader
+  - Decouple ParSetHeader from BlockID (https://github.com/celestiaorg/celestia-core/pull/441)
+  - Stop Signing over the PartSetHeader (https://github.com/celestiaorg/celestia-core/pull/457)
+  - We still don’t want to sign over the PartSetHeader, but we will not be able to use the same  mechanism used in the linked PR, as that way requires decoupling of the PSH from the BlockID
+  - Remove PSH from some consensus messages (https://github.com/celestiaorg/celestia-core/pull/479)
+
+This ADR overrides ADR 005 Decouple BlockID and the PartSetHeader. The PartSetHeader and the BlockID will mostly remain the same. This will make pulling changes from upstream much easier
+
+## Status
+
+Pending review
+
+## Consequences
+
+### Positive
+ - Pulling changes from upstream is streamlined
+ - Separation of functionality will help us iterate faster
+ - Prepare for future designs
+ - Don’t have to have two p2p stacks in a single repo
+### Negative
+ - Perform some computation multiple times
+ - Run multiple nodes instead of a single node
```

Comment:
```suggestion
 - Run multiple nodes instead of a single node (in the case the full celestia-node wants to participate in the consensus protocol)
```